css/properties/appearance.json

```diff
@@ -250,34 +226,50 @@
                   "partial_implementation": true
                 }
               ],
-              "firefox_android": {
-                "version_added": "4",
-                "partial_implementation": true
-              },
+              "firefox_android": [
+                {
+                  "version_added": "80"
+                },
+                {
+                  "version_added": "4",
+                  "partial_implementation": true
+                }
+              ],
               "ie": {
                 "version_added": false
               },
-              "opera": {
-                "version_added": "15"
-              },
+              "opera": [
+                {
+                  "version_added": "69"
+                },
+                {
+                  "version_added": "15",
+                  "partial_implementation": true
+                }
+              ],
               "opera_android": {
-                "version_added": "14"
+                "version_added": "14",
+                "partial_implementation": true
               },
               "safari": {
-                "version_added": "3"
+                "version_added": "3",
+                "partial_implementation": true
               },
               "safari_ios": {
-                "version_added": "1"
+                "version_added": "1",
+                "partial_implementation": true
               },
               "samsunginternet_android": {
-                "version_added": "1.0"
+                "version_added": "1.0",
```

Comment:
If you run `npm run mirror -- --modify=always samsunginternet_android css` you'll get a bunch of updates to apply. One of those ought to be here.

Reply:
This wasn't done, but I'm not going to block the PR on that happening. @AdaRoseCannon FYI that there will be some updating needed here. Some of these changes in these PR will only be possible to mirror when the next version of Samsung Internet, probably 13.1, is in BCD.